components/home/Partners.tsx

```diff
@@ -10,6 +10,7 @@ import { Box, Text } from '@chakra-ui/react'
 const Partners = ({ partnersImages }: PartnersProps) => {
   return (
     <div className="section-my">
+      <h3 className='absolute m-12 mt-[80px] hidden md:block md:ml-[475px] lg:ml-[710px] xl:ml-[950px]  text-xl text-white max-w-lg '>Past and Present</h3>
```

Comment:
Could we use `ml-auto` instead to keep the text aligned to the right?

Reply:
done!